src/swsssdk/dbconnector.py

```diff
@@ -17,23 +18,29 @@ def __init__(self, **kwargs):
 
 SonicV1Connector.db_map = _connector_map[SonicV1Connector.__name__]['db_map']
 
-if len(SonicV1Connector.db_map) != len({v['db'] for k, v in SonicV1Connector.db_map.items()}):
-    raise RuntimeError("Duplicate DB index detected in configuration.")
-
-
 class SonicV2Connector(DBInterface):
     def __init__(self, **kwargs):
         super(SonicV2Connector, self).__init__(**kwargs)
 
+    # after py test cases(e.g snmpagent/tests) are modified
+    # we can do the replacement on this new API via remnaming connect_new to connect
+    # def connect(self, db_name, retry_on=True, isTcpconn=False):
+    def connect_new(self, db_name, retry_on=True, isTcpconn=False):
+        if isTcpconn == False:
+            self.redis_kwargs["unix_socket_path"] = _get_sonic_db_socket(db_name)
+            self.redis_kwargs["host"] = None
+            self.redis_kwargs["port"] = None
+        else:
+            self.redis_kwargs["host"] = _get_sonic_db_hostname(db_name)
+            self.redis_kwargs["port"] = _get_sonic_db_port(db_name)
+            self.redis_kwargs["unix_socket_path"] = None
+        super(SonicV2Connector, self).connect(db_name, retry_on)
+
     pass
 
 
 SonicV2Connector.db_map = _connector_map[SonicV2Connector.__name__]['db_map']
 
-if len(SonicV2Connector.db_map) != len({v['db'] for k, v in SonicV2Connector.db_map.items()}):
-    raise RuntimeError("Duplicate DB index detected in configuration.")
```

Comment:
Why remove?

Reply:
> Why remove?

I added the missing LOGLEVEL_DB(3) , PFC_WD_DB(5), FLEX_COUNTERS_DB(5) into existing database.json which are mainly used for python today.  Since both PFC_WD_DB and FLEX_COUNTER_EB has value 5. This check will raise error. But these check is unnecessary, internally we are using string "DBNAME" as key to index(in SonicV2Connector/DBInterface), duplicated value index is fine,